src/libtls/tls_hkdf.c

```diff
@@ -446,16 +454,22 @@ METHOD(tls_hkdf_t, generate_secret, bool,
 	switch (label)
 	{
 		case TLS_HKDF_C_HS_TRAFFIC:
+			chunk_clear(&this->handshake_traffic_secrets.client);
+			this->handshake_traffic_secrets.client = chunk_clone(okm);
+			/* otherwise fall through to next state */
```

Comment:
```suggestion
			/* fall-through */
```
That's usually enough if no explicit explanation for the fall-through is needed.